<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Permutation Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="exact_checkBox">
     <property name="toolTip">
      <string>If selected, the model is fit to each possible permuation once.
Caution: The number of permuations can blow up quickly with the # of studies and # of moderators</string>
     </property>
     <property name="text">
      <string>exact permuation test (default: no)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>number of iterations for the permutation test when notdoing an exact test</string>
       </property>
       <property name="text">
        <string>Iterations:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="iter_spinBox">
       <property name="toolTip">
        <string>number of iterations for the permutation test when notdoing an exact test (default: 1000)</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string># of decimal places to which to round the printed results</string>
       </property>
       <property name="text">
        <string>Digits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="digits_spinBox">
       <property name="toolTip">
        <string># of decimal places to which to round the printed results</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="retpermdist_checkBox">
       <property name="text">
        <string>get permutation
distribution of test
statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="perm_dist_hist_checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>make histograms
of test statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>retpermdist_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>perm_dist_hist_checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exact_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exact_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>iter_spinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
